queries/vue/highlights.scm

```diff
@@ -1,9 +1,11 @@
 ; inherits: html_tags
 
+(dynamic_directive_inner_value) @tag
```

Comment:
it's not a tag :thinking: If anything, it is something to be injected with js

Reply:
Feel free to push any changes to the PR! That's quicker.